VMware.CloudFoundation.CertificateManagement.psd1

```diff
@@ -10,123 +10,122 @@
 
     # Script module or binary module file associated with this manifest.
     RootModule = '.\VMware.CloudFoundation.CertificateManagement.psm1'
-    
+
     # Version number of this module.
-    ModuleVersion = '1.0.0.1001'
-    
+    ModuleVersion = '1.0.0.1002'
+
     # Supported PSEditions
     # CompatiblePSEditions = @()
-    
+
     # ID used to uniquely identify this module
     GUID = 'ac903c83-c745-44f7-b6bd-1dff113bef14'
-    
+
     # Author of this module
     Author = 'Gary Blake, Ryan Johnson, Kevin Teng, Bhumitra Nagar - Cloud Infrastructure Business Group (CIBG)'
-    
+
     # Company or vendor of this module
     CompanyName = 'VMware Inc.'
-    
+
     # Copyright statement for this module
     Copyright = '(c) 2023 VMware All rights reserved'
-    
+
     # Description of the functionality provided by this module
     Description = 'A PowerShell module for VMware Cloud Foundation certificate management.'
```

Comment:
Remove the full stop from this description, looks better with out

Reply:
Updating to 'PowerShell Module for VMware Cloud Foundation Certificate Management'.

Reply:
Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/pull/15/commits/7569fa212ab76b01d00e451ea281d23a8e49a337.